python/ray/autoscaler/_private/aws/config.py

```diff
@@ -33,28 +33,26 @@
 DEFAULT_RAY_IAM_ROLE = RAY + "-v1"
 SECURITY_GROUP_TEMPLATE = RAY + "-{}"
 
-# V61.0 has CUDA 11.2
-DEFAULT_AMI_NAME = "AWS Deep Learning AMI (Ubuntu 18.04) V61.0"
+DEFAULT_AMI_NAME = "Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911"
 
-# Obtained from https://aws.amazon.com/marketplace/pp/B07Y43P7X5 on 6/10/2022.
```

Comment:
Could you please add a corresponding comment here with the link you got these from?

Reply:
I got those by querying the AWS EC2 API, not from a web page:

```console
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region us-east-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region us-east-2
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region us-west-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region us-west-2
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region ca-central-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region eu-central-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region eu-west-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region eu-west-2
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region eu-west-3
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region sa-east-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region ap-northeast-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region ap-northeast-2
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region ap-northeast-3
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region ap-southeast-1
aws ec2 describe-images --owners amazon --filters "Name=name,Values=Deep Learning Base GPU AMI (Ubuntu 20.04) 20230911" --query "Images[0].ImageId" --output text --region ap-southeast-2
```

Reply:
Hopefully resolved with https://github.com/ray-project/ray/pull/39721/commits/3f4c4247505f9f1018c73ac3cdb3bfd1f60c6bf2